inc/sccmxml.class.php

```diff
@@ -343,9 +376,9 @@ function setNetworks() {
                $CONTENT->addChild('NETWORKS');
                $NETWORKS = $this->sxml->CONTENT[0]->NETWORKS[$i];
 
-               if(filter_var(trim($ip), FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)){
+               if(filter_var(trim($ip), FILTER_VALIDATE_IP, FILTER_FLAG_IPV4)) {
                   $NETWORKS->addChild('IPADDRESS', trim($ip));
-               } else {
+               } else if(filter_var(trim($ip), FILTER_VALIDATE_IP, FILTER_FLAG_IPV6)) {
```

Comment:
CAn you explain ?

why are you forcing IPV6 check ?

Reply:
If IP is IPV4 set IPADDRESS, but is IPV6 set IPADDRESS6, in other case is not an IP.

Before this, if IP is IPV4 set IPADDRESS else IPADDRESS6 is set ... IPADDRESS6 could get invalid value ...

Perhaps this change is for other pull request and other discussion ...